client/state/sites/reducer.js

```diff
@@ -155,8 +155,11 @@ export const items = withSchemaValidation( sitesSchema, ( state = null, action )
 							};
 							break;
 						}
-						case 'wpcom_coming_soon': {
-							const isComingSoon = parseInt( settings.wpcom_coming_soon, 10 ) === 1;
+						case 'wpcom_coming_soon':
+						case 'wpcom_public_coming_soon': {
+							const isComingSoon =
+								parseInt( settings.wpcom_public_coming_soon, 10 ) === 1 ||
+								parseInt( settings.wpcom_coming_soon, 10 ) === 1;
```

Comment:
Will launching a site remove wpcom_coming_soon option as well as wpcom_public_coming_soon site options?

Reply:
My first response, without thinking too deeply, would be `YES`. 

If we're turning off coming soon for everyone as soon as they launch without the possibility of turning it back on (controversial) then yes. 

Also, we don't want any regression to v1 so when and if we do allow folks to toggle coming soon mode on and off we'll want them to use `wpcom_public_coming_soon` from v2 and cast v1 options into the fiery pit of oblivion.

Do you see anything missing here that doesn't take that into account? 🤔